### DISCUSSION TITLE
Where is gone beta nextjs 13 documentation?

### DISCUSSION THREAD

**Discussion by shavidze** [vercel/next.js#49306] [Help]
### Summary

Where is gone beta nextjs 13 documentation?

**Comment by icyJoseph** [comment]
It is all into the same page now.

![Screenshot 2023-05-05 at 15 44 24](https://user-images.githubusercontent.com/21013447/236474819-c8717c4c-d89f-43f2-ac94-ecf65b6b987c.png)


Use the dropdown to select which version you want to see. App or Pages dir.

**Reply by shavidze** [reply]
I see, but there are many topics that were in beta docs but I can't see there

**Reply by icyJoseph** [reply]
Is it related to deprecated stuff? Like the old head.tsx files?

**Reply by shavidze** [reply]
There was a post about Server and Client Components in Rendering section and lots of other stuffs that I can't see anymore
